music_assistant/server/providers/radiobrowser/__init__.py

```diff
@@ -94,15 +89,15 @@ async def search(
         result = SearchResults()
         if MediaType.RADIO not in media_types:
             return result
+        
+        search_result = await self.radios.search(name=search_query, limit=limit)
 
-        searchresult = await self.radios.search(name=search_query, limit=limit)
-
-        for item in searchresult:
+        for item in search_result:
             result.radio.append(await self._parse_radio(item))
 
         return result
 
-    @use_cache(86400 * 7)
+    @use_cache(60)
```

Comment:
why ? lets be careful with doing many requests. Is it really true that the output of the radiobrowser api changes often ? Maybe make this a 1 hour cache at least, not 60 seconds!

Reply:
True we can increase it, we can even do one day but 7 days in my opinion is a bit too much